kitodo-mediaserver-importer/src/test/java/org/kitodo/mediaserver/importer/processors/WorkDataReaderTest.java

```diff
@@ -0,0 +1,110 @@
+package org.kitodo.mediaserver.importer.processors;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.kitodo.mediaserver.core.db.entities.Work;
+import org.kitodo.mediaserver.core.processors.SimpleList2MapParser;
+import org.kitodo.mediaserver.core.processors.XsltMetsReader;
+
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.util.ResourceUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+/**
+ * Unit tests for the work data reader.
+ */
+@RunWith(SpringRunner.class)
+public class WorkDataReaderTest {
+
+    private XsltMetsReader xsltMetsReader = new XsltMetsReader();
+    private WorkDataReader workDataReader = new WorkDataReader();
+    private SimpleList2MapParser readResultParser = new SimpleList2MapParser();
+
+    @Before
+    public void init() throws Exception {
+        File xsltFile  = ResourceUtils.getFile("classpath:xslt/getWorkData.xsl");
+        xsltMetsReader.setXslt(xsltFile);
+        workDataReader.setMetsReader(xsltMetsReader);
+        readResultParser.setMapSeparator(":");
+        workDataReader.setReadResultParser(readResultParser);
+    }
+
+    @Test(expected = FileNotFoundException.class)
+    public void exceptionThrownWhenFileNotPresent() throws Exception {
+        File testMetsFile  = ResourceUtils.getFile("classpath:metsfiles/notHere.xml");
+
+        workDataReader.read(testMetsFile);
+    }
+
+    @Test
+    public void generatesWorkObject() throws Exception {
+        // given
+        File testMetsFile  = ResourceUtils.getFile("classpath:metsfiles/singleIdentifierMets.xml");
+
+        // when
+        Work workData = workDataReader.read(testMetsFile);
+
+        // then
+        assertThat(workData).isInstanceOf(Work.class);
+        assertThat(workData.getId()).isNotNull();
+    }
+
+    @Test
+    public void includesMultipleIdentifiers() throws Exception {
+        // given
+        File testMetsFile  = ResourceUtils.getFile("classpath:metsfiles/multipleIdentifiersMets.xml");
+
+        // when
+        Work workData = workDataReader.read(testMetsFile);
+
+        // then
+        assertThat(workData).hasFieldOrProperty("identifiers");
+        assertThat(workData.getIdentifiers().size() > 1).isTrue();
+    }
+
```

Comment:
It should be testet not only that the file includes multiple identifiers, but also that they are correctly read (check identifier and type values as well).